dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -104,3 +104,7 @@ enableImplicitBeanArchives.conflict.CWOWB1017W.useraction=Update your server.xml
 unversioned.beansXML.warning.CWOWB1018W=CWOWB1018W: Using an unversioned CDI beans.xml file might lead to unexpected behaviour when using CDI 4.0 or later: {0}
 unversioned.beansXML.warning.CWOWB1018W.explanation=Prior to CDI 4.0, a non-empty but unversioned beans.xml file meant that the archive was treated as an explicit bean archive (all classes are eligible to be beans). In CDI 4.0 or later, the archive is taken to be an implicit bean archive (only annotated classes are eligible be beans).
 unversioned.beansXML.warning.CWOWB1018W.useraction=Update the beans.xml file within your application to use a newer version of the schema and ensure that the version and bean-discovery-mode attributes are properly set.
+
+no.bda.for.module.CWOWB1019E=CWOWB1019E: WebSphere Application Server internal error occurred. Please contact WebSphere Application Server support with the following data: {0} {1}
+no.bda.for.module.CWOWB1019E.explanation=CMSG0001
+no.bda.for.module.CWOWB1019E.useraction=See the problem determination information on the WebSphere Application Server Support web page: http://www.ibm.com/software/webservers/appserv/was/support.
```

Comment:
This message doesn't look great, but maybe it matches existing messages?

In this case it certainly is an internal error and not something the user should have to deal with.

Reply:
Its a copy of CWOWB1003E - CWOWB1006E. Isn't that what we agreed on in the call?

Reply:
We didn't say what it was a copy of on the call and I can't tell where you got it from when I'm looking at the diff.

The language here is contrary to the current message guidelines (likely because the place you copied it from is old) so it's not what I expected.

Reply:
In other features, we just have one general "An internal exception occurred" message which is meant to have the exception appended.

In CDI, it looks like we've got a separate identical message for each different internal error case (as can be seen further up in the file) so, although it doesn't follow the current message guidelines, this message is consistent with existing messages for this component.